### ISSUE TITLE
could not understand ‘backend.tf.nn.swish(x)’

### ISSUE THREAD

**Issue by wangyexiang** [qubvel/efficientnet#83]
![图片](https://user-images.githubusercontent.com/23202109/69144867-46e04180-0b07-11ea-82d1-6ec6e68d1e25.png)
Dear sir, it seems that ‘backend.tf.nn.swish(x)’ is not carried out forever.

**keras version:2.3.1
backend:tensorflow2.0-cpu**

**Comment by shaunster0** [comment]
having same problems with swish, even when I changed to x * sigmoid(x)
